packages/transport/test/testHelpers/TestUtil.ts

```diff
@@ -135,11 +135,7 @@ export class TestUtil {
         });
     }
 
-    public static async expectThrowsAsync(method: Function | Promise<any>, customExceptionMatcher: (e: Error) => void): Promise<void>;
-
-    public static async expectThrowsAsync(method: Function | Promise<any>, errorMessageRegexp: RegExp | string): Promise<void>;
-
-    public static async expectThrowsAsync(method: Function | Promise<any>, errorMessageRegexp: RegExp | string | ((e: Error) => void)): Promise<void> {
+    private static async expectThrowsAsync(method: Function | Promise<any>, errorMessageRegexp: RegExp | string | ((e: Error) => void)): Promise<void> {
```

Comment:
why is this expectThrowsAsync not removed?

Reply:
Because it is used in `expectThrowsRequestErrorAsync` to check the correct error code.

Reply:
We could use .rejects there too, or not? Or at least move the logic inside the other function.

Reply:
I just saw you made it async ... then it's okayish.